tracing/error_reporter_hooks.go

```diff
@@ -1,11 +1,6 @@
 package tracing
 
-import "github.com/reddit/baseplate.go/log"
-
-const (
-	success = "success"
-	fail    = "fail"
-)
+import "github.com/getsentry/raven-go"
```

Comment:
style nit: for single import, still use `()`:

```
import (
  "github.com/getsentry/raven-go"
)
```

as this makes future diff cleaner.